gapic/src/main/com/google/gapid/views/FramebufferView.java

```diff
@@ -133,6 +133,11 @@ private RenderSetting(int maxWidth, int maxHeight, Path.DrawMode drawMode) {
   private final AttachmentPicker picker;
   private final Label pickerLabel;
 
+  private final GridData pickerGridData;
+  private final SashForm splitter;
```

Comment:
It looks like the splitter is only accessed locally at initialization, and doesn't need to be member variable?